common/include/pcl/impl/point_types.hpp

```diff
@@ -822,22 +828,30 @@ namespace pcl
     PCL_MAKE_ALIGNED_OPERATOR_NEW
   };
 
-
-
   PCL_EXPORTS std::ostream& operator << (std::ostream& os, const PointXY& p);
   /** \brief A 2D point structure representing Euclidean xy coordinates.
     * \ingroup common
     */
-  struct PointXY
-  {
-    float x = 0.f;
-    float y = 0.f;
-
+  struct EIGEN_ALIGN16 PointXY
```

Comment:
I am not sure about the alignment. Each PointXY is only 2*4=8 bytes large, if each point gets aligned to 16 bytes, then they take twice as much space as necessary. Or is the alignment needed for the vector map?

Reply:
I guess its necessary for the vectormap